skyve-war/src/main/java/modules/admin/User/UserService.java

```diff
@@ -267,4 +274,126 @@ public void evictUserProxy(UserExtension bean) {
 			p.evictSharedCachedBean(UserProxy.MODULE_NAME, UserProxy.DOCUMENT_NAME, bizId);
 		}
 	}
+
+	/**
+	 * Returns the current session/conversation user as an Admin module User
+	 *
+	 * @return The current {@link modules.admin.User.UserExtension}
+	 */
+	@SuppressWarnings("static-method")
+	public UserExtension currentAdminUser() {
+		UserExtension result = null;
+		try {
+			Persistence p = CORE.getPersistence();
```

Comment:
Persistence should be injected in service classes. Then the method should not need the static method warning supression.